scalardl/project.clj

```diff
@@ -10,12 +10,12 @@
                  [cc.qbits/hayt "4.1.0"]]
   :profiles {:dev {:dependencies [[tortue/spy "2.0.0"]]
                    :plugins [[lein-cloverage "1.1.2"]]}
-             :use-released {:dependencies [[com.scalar-labs/scalardl-java-client-sdk "3.0.1" :exclusions [org.slf4j/slf4j-log4j12]]]}
+             :use-released {:dependencies [[com.scalar-labs/scalardl-java-client-sdk "3.3.1" :exclusions [org.slf4j/slf4j-log4j12]]]}
```

Comment:
Not fully sure but it seems like we use the master for the Ledger, does it?
If so, does it work?

Reply:
> Not fully sure but it seems like we use the master for the Ledger, does it?
If so, does it work?

Yes, It works fine without changing the version of `scalardl-java-client-sdk`.

Do we need to remove the version update?